docs/source/usage/keylime.rst

```diff
@@ -44,6 +45,22 @@ Modify /etc/keylime.conf:
     [registrar]
     registrar_ip = 0.0.0.0
 
+
```

Comment:
Can you add a subsection header here - something like

Boot Measurement Setup
~~~~~~~~~~~~~~~~~~~

Reply:
Actually, don't do that, because there are steps after this that apply to all uses, regardless of boot attestation - maybe put some of the subsequent steps in list format?

Reply:
Actually, don't do that, because there are steps after this that apply to all uses, regardless of boot attestation - maybe put some of the subsequent steps in list format?

Reply:
what needs to be in a list here exactly?

Reply:
The goal in general is to not have a wall of text; here specifically because it's part of an optional series of steps. We want to delineate the section here that is optional in a way that it's easy for the reader to immediately parse.

I'd actually suggest capitalizing "Provision the node", "Install Keylime packages", "Modify /etc/keylime.conf". You'll see that each of these new marked sections will have a series of steps in list format.

Then make your section a capitalized "(Optional) Enable Boot measurement attestation". Put the steps necessary to do so in a list below it.

Then capitalize "Start Keylime services".

Reply:
Ive made some changes